src/NuGet.Clients/NuGet.PackageManagement.UI/ViewModels/PackageItemViewModel.cs

```diff
@@ -147,7 +147,7 @@ public NuGetVersion LatestVersion
                         var displayVersion = new DisplayVersion(_latestVersion, string.Empty);
                         string toolTipText = PackageLevel == PackageLevel.Transitive ? Resources.ToolTip_TransitiveDependencyVersion : Resources.ToolTip_LatestVersion;
                         LatestVersionToolTip = string.Format(
-                            CultureInfo.CurrentUICulture,
+                            CultureInfo.CurrentCulture,
```

Comment:
This should be left unchanged, as it is a user-facing string resource.

```suggestion
                            CultureInfo.CurrentUICulture,
```

Reply:
The advice we got over Teams was "If you are doing resource lookup, at that time the CurrentUICulture is what is correct to use", but otherwise CurrentCulture is correct.

In this case, the previous line, line 148, `string toolTopText = ... Resources.Name ...` is doing the resource lookup. Here in the `string.Format`, it's not doing a string lookup, and therefore CurrentCulture is correct.

Reply:
Agree with Andy. You can omit my previous comments.